app/javascript/components/App.js

```diff
@@ -27,6 +26,21 @@ const App = (props) => {
       .catch((error) => console.log(error))
   }
 
+  const createApartment = (apartment) => {
+    console.log(apartment)
+    // fetch("http://localhost:3000/apartments", {
+    //   body: JSON.stringify(apartment),
+    //   headers: {
+    //     "Content-Type": "application/json"
+    //   },
+    //   method: "POST"
+    // })
+    // .then((response) => response.json())
+    // .then((payload) => readApartments())
+    // .catch((errors) => console.log("Apartment create errors:", errors))
+  }
```

Comment:
If we are going to bring this in let's just uncomment it and knock out both cards!